int32 type
int32 value
string mapName
---
int32 returnValue
string returnString